java/src/main/java/vw/jni/NativeUtils.java

```diff
@@ -88,7 +84,7 @@ else if (osName.toLowerCase().contains("linux")) {
             if (version == null) {
```

Comment:
I think we should add a check for the period as well.  So the whole thing should read

```
int separatorIndex = version.indexOf(".")
if (separatorIndex < 0) {
    throw new UnsupportedOperationException("Cannot find separator of . in linux version of " + version);
}
return distro + version.subString(0, separatorIndex + 1);